### ISSUE TITLE
[Bug] No validation performed on phone number

### ISSUE THREAD

**Issue by pierreTklein** [hacknyu/website#135]
User can enter in any number of characters and numbers into the emergency contact information.

![image](https://user-images.githubusercontent.com/17463268/52583950-d6312980-2dfe-11e9-98be-f70081cc05a0.png)

**Comment by NicholasLYang** [comment]
Yeah in general our validation is pretty crappy. This is partially why we want to switch off of firebase and onto a backend that is a little more traditional.

**Comment by pierreTklein** [comment]
Similar issue for birthdate:

![image](https://user-images.githubusercontent.com/17463268/52584041-17c1d480-2dff-11e9-9b33-68497796116c.png)

**Comment by pierreTklein** [comment]
Fair, fair. you could do client-side validation for now. Look into `Formik` for form creation, and `Yup` for validation :)

**Comment by NicholasLYang** [comment]
We're already using react final form which comes with validation, it's honestly more a matter of getting around to writing the validation code lol.
